src/apps/webhooks.clj

```diff
@@ -13,8 +13,13 @@
                       (fields [:wt.topic])
                       (where {:ws.webhook_id webhook-id}))))
 
-(defn- format-webhook [{:keys [id] :as webhook}]
-  (assoc webhook :topics (get-webhook-topics id)))
+(defn- get-webhook-type [type]
+  (first (select [:webhooks_type :wt] (where {:wt.type type}))))
```

Comment:
This doesn't have to be changed, but a table alias isn't really necessary when you're not doing any joins. For example, you could change this function to something like this:

```clojure
(defn- get-webhook-type [type]
  (first (select :webhooks_type (where {:type type}))))
```

The alias doesn't do any harm, of course. It's just that leaving it off saves a little bit of typing. 😄

Reply:
👍